pygeoapi/process/manager/__init__.py

```diff
@@ -26,3 +28,36 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 #
 # =================================================================
+
```

Comment:
I see that other base classes doesn't use to put something in the `__init__`. Maybe, just for consistency, we can move this into the `base.py` file